pootle/templates/includes/formtable.html

```diff
@@ -24,11 +24,7 @@
 		  {% endblocktrans %}
 	      </span>
 	      <span class="pagination-info js-pagination-items-per-page">
-          {% blocktrans count n=formtable.results_per_page trimmed %}
-            Showing {{ n }} result per page
-          {% plural %}
-            Showing {{ n }} results per page
-          {% endblocktrans %}
+		{{ formtable.msg_results_per_page }} {{ formtable.results_per_page }}
```

Comment:
What about languages that have the number at the beginning of the string. Or languages where the number appears in the middle of the string?